packages/common/src/tokenizer.test.ts

```
@@ -0,0 +1,106 @@
+import { describe, it, expect, vi, beforeEach } from 'vitest';
+
+const mockTokenize = vi.fn();
+
+vi.mock('kuromoji', () => ({
+  default: {
+    builder: vi.fn(() => ({
+      build: vi.fn((cb: (err: Error | null, tokenizer: any) => void) => {
```

ESLint

[error] 'err' is defined but never used.

(no-unused-vars)